cdap-app-fabric/src/main/java/co/cask/cdap/metadata/MetadataHttpHandler.java

```diff
@@ -224,29 +227,52 @@ public void searchMetadata(HttpRequest request, HttpResponder responder,
                              @QueryParam("showHidden") @DefaultValue("false") boolean showHidden,
                              @QueryParam("showCustom") boolean showCustom,
                              @Nullable @QueryParam("entityScope") String entityScope) throws Exception {
+    SearchRequest searchRequest = getValidatedSearchRequest(namespaceId, searchQuery, targets, sort, offset,
+                                                            limit, numCursors, cursor, showHidden, entityScope);
+
```

Comment:
can we trace log the search request here

Reply:
done